lms/static/js/ccx/schedule.js

```diff
@@ -157,7 +180,9 @@ var edx = edx || {};
               return !node.hidden;});
             this.$el.html(schedule_template({chapters: this.showing}));
             $('table.ccx-schedule .sequential,.vertical').hide();
-            $('table.ccx-schedule .unit .toggle-collapse').on('click', this.toggle_collapse);
+            $('table.ccx-schedule .unit .toggle-collapse').off('click').on(
```

Comment:
I don't understand why this change was necessary. It looks like you're removing all the "click" event handlers before adding a new one. Is a "click" event handler getting attached somewhere and forgotten about? It seems like it would be better to figure out what's going on, rather than just stripping all the event handlers like this.